test/message-test.js

```diff
@@ -148,9 +148,10 @@ describe("Message API", function () {
 				.reply(200, messagesList, {
 					"link" : "<https://api.catapult.inetwork.com" +
 						"/v1/users/" + userId + "/messages?fromDateTime=" +
-						fromDateTime + "&" + "toDateTime=" + toDateTime + ">"
```

Comment:
Did we decide not to support querying by date and time at all through the SDK? If we support this API function, maybe we should keep this test, and the sortKey test should be a separate test?